epubmaker_postprocessing.rb

```diff
@@ -1,10 +1,11 @@
 require 'fileutils'
 require 'net/smtp'
 
-unless defined?(ENV['TRAVIS_TEST'])
+unless (ENV['TRAVIS']) == 'true'
```

Comment:
It is:  https://docs.travis-ci.com/user/environment-variables#Default-Environment-Variables  We can put one back in the .yml if you'd rather?